sdks/common/fn-api/src/test/resources/org/apache/beam/fn/v1/standard_coders.yaml

```diff
@@ -62,6 +62,18 @@ coder:
   urn: "beam:coders:kv:0.1"
   components: [{urn: "beam:coders:bytes:0.1"},
                {urn: "beam:coders:bytes:0.1"}]
+nested: false
+examples:
+  "\u0003abcdef": {key: abc, value: def}
+  "\u0004ab\0cde\0f": {key: "ab\0c", value: "de\0f"}
+
+---
+
+coder:
+  urn: "beam:coders:kv:0.1"
```

Comment:
FWIW a URN technically needs to start with "urn" (otherwise this is a URI with the scheme `beam`). Maybe that is implicit here.